adoc/admin-security-certificates.adoc

```diff
@@ -0,0 +1,208 @@
+= Certificates
+
+During the installation of {productname}, a CA (Certificate Authority) certificate is generated; that is then used to authenticate and verify all communications. The process also creates and distributes client certificates for the components.
+
+Communication is secured with TLS v1.2 using the AES 128 CBC cipher. All client certificates are 4096 Bit RSA encrypted.
+
+The CA certificate is valid for 3650 days (10 years) by default. The client certificates are valid for 365 days (1 year) by default.
+
+Certificates are located in `/etc/kubernetes/pki` on each master nodes. Three root CA required for {productname} are
+
+* {kube} apiserver: path `/etc/kubernetes/pki/ca.{crt.key}`
+* front-end proxy: path `/etc/kubernetes/pki/front-proxy-ca.{crt.key}`
+* etcd: path `/etc/kubernetes/pki/etcd/ca.{crt.key}`
+
+The following certificates are managed by `kubeadm`.
+
+[%header,cols=4*]
+|===
+|Common Name
+|Parent CA
+|Path
+|Kind
+
+|kubernetes
+|
+|ca.{crt.key}
```

Comment:
I assume you want to describe ca.crt, ca.key? why not just ca.crt, ca.key.

Reply:
Cuz like apiserver-etcd-client.crt, apiserver-etcd-client.key would too long.
How about follows [k8s doc format](https://kubernetes.io/docs/setup/best-practices/certificates/#single-root-ca)?